proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,535 @@
+   Mitglied mit schriftlicher Vollmacht vertreten lassen.
+7. Die Teilnahme an der Mitgliederversammlung und die Stimmabgabe müssen auch auf
+   elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann eine
+   Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.
+
+## 7. Aufgaben der Mitgliederversammlung
+
+1. Die Generalversammlung ist das höchste beschlussfassende Organ und ist
+   für alle Aufgaben zuständig, sofern diese nicht anderen zugewiesen
+   wurden.
+2. Sie wählt den Vorstand aus dem Kreis der Kernmitgliedern und kann
+   Vorstandsmitglieder abwählen.
+3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
+4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
+   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
+5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
+6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
+   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
+   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
+   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
+   beim Verein angestellt ist, um die Buchführung einschließlich Jahresabschluss
+   zu prüfen und über das Ergebnis vor der Mitgliederversammlung zu berichten.
+   Dr Rechnungsprüfer hat Zugang zu allen Buchungs- und Rechnungsunterlagen des
+   Vereins.
+7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
+   Krediten.
+
+## 8. Der Vorstand
+
+1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
+   Schatzmeister die für eine Amtszeit von 3 Jahren gewählt werden.
+
+   TODO: Definieren Sie die Verantwortlichkeiten der Rollen.
+
+2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen sind wird durch eine
+   Geschäftsordnung des Vorstands geregelt.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die Beschlüsse
+   der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand auf Basis einer Internetwahl ein kommissarisches
+   Vorstandsmitglied. Das kommissarische Vorstandsmitglied bleibt bis zur
+   nächsten Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
+7. Der Vorstand kann zur Führung der laufenden Geschäfte einen hauptamtlichen
+   Geschäftsführer bestellen.
+8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
+   Vorstandssitzungen teil.
+9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
```

Comment:
... ohne Mitgliederabstimmung / eigenständig ... ?

Reply:
Yes, for my understanding this is redundant with the now removed Severability Clause. 
It is only for authorities demand regarding the tax status. Here a formal process of a General Assembly is to long. I will add that the general assembly needs to confirm this change later.

Reply:
@ronso0 is this OK now?